src/Commands/Env/CodeLogCommand.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Env;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Terminus\Collections\Sites;
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+
+class CodeLogCommand extends TerminusCommand
+{
+    /**
+     * Contructor that supports injection for unit testing.
+     */
+    public function __construct($sites = null)
+    {
+        parent::__construct();
+        $this->sites = $sites ? $sites : new Sites();
+    }
+
+    /**
+     * Show an environment's code log.
+     *
+     * @command env:code-log
+     *
+     * @param string $site_env Site & environment to show log for.
+     *
+     * @usage terminus env:code-log my-site.dev
+     *   Show code log for the `dev` environment for site `my-site`.
+     */
+    public function code_log($site_env)
+    {
+        list($site_name, $env_name) = explode('.', $site_env);
+        $site = $this->sites->get($site_name);
+        $env  = $site->environments->get($env_name);
+
+        $logs = $env->commits->all();
+        $data = [];
+        foreach ($logs as $log) {
+            $data[] = [
+                'time'    => $log->get('datetime'),
+                'author'  => $log->get('author'),
+                'labels'  => implode(', ', $log->get('labels')),
+                'hash'    => $log->get('hash'),
+                'message' => trim(
+                    str_replace(
+                        "\n",
+                        '',
+                        str_replace("\t", '', substr($log->get('message'), 0, 50))
```

Comment:
Yeah maybe I just copied it from the old code... :) We're porting the 0.x commands to 1.0